ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Offset.scala

```diff
@@ -21,22 +20,18 @@ import com.google.protobuf.ByteString
   * less than newer offsets.
   *
   */
-final class Offset(private val value: ByteString) extends AnyVal with Ordered[Offset] {
+sealed trait OffsetTag
 
-  override def compare(that: Offset): Int =
-    Offset.comparator.compare(value, that.value)
+object Offset {
 
-  def toByteArray: Array[Byte] = value.toByteArray
+  val tag = Tag.of[OffsetTag]
```

Comment:
We are not using tagged types in participant state yet, and I'm not sure we should start now tbh.

Reply:
I change it to case class